classes/shop/ShopUrl.php

```diff
@@ -189,6 +189,34 @@ public static function getShopUrls($idShop = false)
     }
 
     /**
+     * Get the default shop URL, which is the first defined main URL.
+     *
+     * @return bool|ShopUrl ShopUrl object or false on failure to find one.
+     *
+     * @since   1.1.0
+     * @version 1.1.0 Initial version
+     */
+    public static function getDefaultUrl()
+    {
+        global $shopUrlConfig;
```

Comment:
Not sure if a global is the way to go then. Let's see if we can find an alternative first.